docs/sites/faq.rst

```diff
@@ -3,3 +3,28 @@ FAQ
 
 Whenever you find out something that you were surprised by or needed some non-trivial
 thinking, please add it here.
+
+How do I define additional unique keys in a ``dy.Schema``?
+----------------------------------------------------------
+
+By default, ``dataframely`` only supports defining a single non-nullable primary key in ``dy.Schema``.
```

Comment:
Since nomenclature around primary keys is slightly fuzzy, I want to make it clear here that dataframely _does_ support composite primary keys. How about adjusting this slightly:

```suggestion
By default, ``dataframely`` only supports defining a single non-nullable (composite) primary key in ``dy.Schema``.
```